.github/scripts/generate_binary_build_matrix.py

```diff
@@ -24,6 +25,7 @@
 
 ROCM_ARCHES = ["6.0", "6.1"]
 
+XPU_ARCHES = ["xpu"]
```

Comment:
@chuanqi129 separate this PR into 2.
-  First build triton-xpu wheel. The changes in this file and [generated-linux-binary-manywheel-nightly.yml](https://github.com/pytorch/pytorch/pull/129560/files#diff-7296837255a68baadd9f386ec756ab581773111df14bc37752d68f53c7b36d79) are not required at this point
- Second PR actually add manywheel build.

Reply:
Thanks @atalman. Make sense, have split the triton-xpu wheel build in PR #129730.